### ISSUE TITLE
Default Alt Text Inline guidance to open

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11697]
## Description
We implemented inline guidance for the alt text field at the beginning of October.  This component is currently defaulted to closed and the additional guidance is hidden from editors when first loaded.  Based on a recent audit, this pattern has not significantly improved the alt text that editors are adding to their images.

We would like to change the default behavior to open so that the additional guidance is expanded and readily visible and available to editors. Once completed we will run another audit to see what impact (if any) this pattern may have had on the alt text editors are adding.

Note: the default behavior for the inline guidance component should be able to be set separately for each instance of the component. For example, the alt text guidance may be defaulted to open but we may use this on the Paths field and default it to closed.

## Acceptance Criteria
- [ ] Change default behavior for alt text inline guidance component to open
- [ ] Ensure that the default behavior can be set separately for each instance of the component
- [ ] QA Review

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`
